website/docs/sdk-reference/react.mdx

```diff
@@ -977,6 +987,48 @@ If you do not want to expose the SDK key or the content of the config JSON file,
 
 Also, we recommend using [confidential targeting comparators](../targeting/targeting-rule/user-condition.mdx#confidential-text-comparators) in the Targeting Rules of those feature flags that are used in the frontend/mobile SDKs.
 
+## Using multiple providers
+
+To access a feature flag's value, your code needs to refer to that Config where that particular feature flag is located. This is what a ConfigCatProvider does for you; it refers to the Config by specifying the Config's sdkKey.
+
+If your code needs access to feature flags from multiple Configs, you will need multiple ConfigCatProviders - each pointing to a Config by specifying that particular Config's sdkKey.
+
+And when your code wants to access a particular feature flag, it should refer to the specific ConfigCatProvider that is associated with the Config where that particular feature flag is located. This is exactly what the providerId allows your code to do: to refer to the specific ConfigCatProvider that is associated with the feature flag it wants to access.
+
+```tsx
+const CC_PROVIDERID = {
+  COMMON: "COMMON",
+  FRONTEND: "FRONTEND"};
```

Comment:
```suggestion
const CC_PROVIDERID = {
  COMMON: "COMMON",
  FRONTEND: "FRONTEND"
} as const;
```

See also: https://maxheiber.medium.com/alternatives-to-typescript-enums-50e4c16600b1#ecf7